library/foundation/src/main/java/com/epmedu/animeal/foundation/theme/Theme.kt

```diff
@@ -43,7 +47,20 @@ fun AnimealTheme(
     MaterialTheme(
         colors = colors,
         typography = Typography,
-        shapes = Shapes,
-        content = content,
-    )
+        shapes = Shapes
+    ) {
+        LaunchedEffect(Unit) {
```

Comment:
Maybe better to split setup systemUiController out of Theme

Reply:
Where? In MainActivity?

Reply:
1. Leave `Surface` inside theme
2. Remove `Surface` from @Preview
3. Extact `systemUiController` into separate `Composable`

Reply:
Done